### PR TITLE
[Merge Conflicts] Get/Set AzureRmRecoveryServices changes

### PR DESCRIPTION
**Pull request by vishak-ms**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishak-ms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: src/Common/Commands.Common/Commands.Common.csproj [hunk 1]
```diff
@@ -189,4 +190,11 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
```
Comment (AditTulasi) [inline src/Common/Commands.Common/Commands.Common.csproj]: Do not check-in nugget restore changes in csproj.

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
@vishak-ms please pull from upstream/dev and resolve conflicts

**Comment by markcowl** [comment]
@vishak-ms there are merge conflicts, please pull the latest and update the PR

**Comment by markcowl** [comment]
@vishak-ms we will close this PR by EOW if you do not update it to remove merge conflicts and pass the ci build.

**Pull request merged**